backend/middleware/src/MicroserviceHandler.js

```diff
@@ -47,8 +47,12 @@ module.exports = class MicroserviceHandler {
 
       res.status(200).send('Successfully completed request ');
     });
-  }
 
+    this.app.post('/live_news', (req, res) => {
+      const { service, news, timespan } = req.body;
```

Comment:
timespan never used:

```suggestion
      const { service, news } = req.body;
```